local_pathfinding/ompl_path.py

```diff
@@ -27,9 +35,19 @@ def __init__(self, parent_logger: RcutilsLogger, max_runtime: float):
         #     simple_setup.simplifySolution()
 
     def get_cost(self):
+        """Returns the cost of the path generated
+
+        Raises:
+            NotImplementedError: Method or function hasn't been implemented yet
+        """
         raise NotImplementedError
 
-    def get_waypoints(self):
+    def get_waypoints(self) -> list[tuple[float, float]]:
+        """Returns a list of waypoints for the boat to follow
+
+        Returns:
+            list: A list of tuples representing the x and y coordinates of the waypoints
```

Comment:
Can add that it returns an empty list and prints a warning message if the path is not solved